src/Compilers/CSharp/Portable/Binder/FieldKeywordBinder.cs

```diff
@@ -1,4 +1,4 @@
-// Licensed to the .NET Foundation under one or more agreements.
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
The change here is *adding* BOM character. Visual Studio added it automatically for me.
The file was originally created in VS Code, so it seems that VS Code doesn't respect this:

https://github.com/dotnet/roslyn/blob/5e4f8781bfaa29ede55a08b978cadde3e0087ffc/.editorconfig#L15